src/frontend/metrics-charts.js

```
@@ -0,0 +1,830 @@
+                        x: { display: false }
+                    }
+                }
+            });
+            break;
+            
+        case 'multi-metric':
+            // Normalize all metrics to 0-100 scale for comparison
+            const normalize = (arr) => {
+                const max = Math.max(...arr);
+                return max > 0 ? arr.map(v => (v / max) * 100) : arr;
+            };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)